pkg/math/cosmos_uint.go

```diff
@@ -0,0 +1,21 @@
+package math
+
+import "cosmossdk.io/math"
```

Comment:
Is `Uint64()` from cosmossdk implemented as arbitrary-precision? Similar to big ints from math/big.
If it's not we would have to check for overflows as Add() could return a panic when trying to return a number that is out of bounds.

Reply:
it's a wrapper for `*big.Int`